docs/portal/image-registry/managing-tags-under-a-repository.md

```diff
@@ -0,0 +1,14 @@
+# Managing Tags Under a Repository
+
+A project can have one or more repositories and a repository can have one or more tags.
+
+The following procedure describes how to view tags for a repository, how to view tag details, and how to delete a tag.
+
+1. In the left pane, click **Image Registry > Registry Projects**.
+2. On the Registry Projects page, click the name of a project. The details page appears.
+3. To view details about a repository's tag:<ul><li>Click a number in the **Tags** column within the **Repository** area at the bottom of the page.</ul></li>
+![null](</docs/resources/images/registry/edit-project-tags.png>)<ul><li>When the next page appears, click a link in the **Tags** column:</ul></li>
```

Comment:
Please change "Tags column" to "Tag column".

Reply:
Fixed, thank you.